terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -0,0 +1,13 @@
+environment = "staging"
```

Comment:
In here you should add the service scale up cron job tasks like this: https://github.com/companieshouse/presenter-account-api/blob/main/terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars